docs/legal/privacy.md

```diff
@@ -1,5 +1,5 @@
 # Privacy Policy
-Last updated: 05 October 2022
+Last updated: 13 October 2022
```

Comment:
All good. Just make sure to update the `Last updated` thing when modifying ToS or Privacy Policy.